docs/tutorials.md

```diff
@@ -12,8 +12,9 @@ Code example:
 import WebTorrent from 'webtorrent'
 
 const client = new WebTorrent()
-const torrentId = "magnet:?xt=urn:btih:08ada5a7a6183aae1e09d831df6748d566095a10&dn=Sintel&tr=udp%3A%2F%2Fexplodie.org%3A6969&tr=udp%3A%2F%2Ftracker.coppersurfer.tk%3A6969&tr=udp%3A%2F%2Ftracker.empire-js.us%3A1337&tr=udp%3A%2F%2Ftracker.leechers-paradise.org%3A6969&tr=udp%3A%2F%2Ftracker.opentrackr.org%3A1337&tr=wss%3A%2F%2Ftracker.btorrent.xyz&tr=wss%3A%2F%2Ftracker.fastcast.nz&tr=wss%3A%2F%2Ftracker.openwebtorrent.com&ws=https%3A%2F%2Fwebtorrent.io%2Ftorrents%2F&xs=https%3A%2F%2Fwebtorrent.io%2Ftorrents%2Fsintel.torrent"
+const torrentId = 'magnet:?xt=urn:btih:08ada5a7a6183aae1e09d831df6748d566095a10&dn=Sintel&tr=udp%3A%2F%2Fexplodie.org%3A6969&tr=udp%3A%2F%2Ftracker.coppersurfer.tk%3A6969&tr=udp%3A%2F%2Ftracker.empire-js.us%3A1337&tr=udp%3A%2F%2Ftracker.leechers-paradise.org%3A6969&tr=udp%3A%2F%2Ftracker.opentrackr.org%3A1337&tr=wss%3A%2F%2Ftracker.btorrent.xyz&tr=wss%3A%2F%2Ftracker.fastcast.nz&tr=wss%3A%2F%2Ftracker.openwebtorrent.com&ws=https%3A%2F%2Fwebtorrent.io%2Ftorrents%2F&xs=https%3A%2F%2Fwebtorrent.io%2Ftorrents%2Fsintel.torrent'
 const player = document.querySelector('video')
+const scope = './'
```

Comment:
I'd prefer this inlined

Reply:
I left this out like this, in case someone runs in a sub-scope like say github pages which would be for example `https://thaunknown.github.io/webtorrent_app/` where the scope would be changed to `./webtorrent_app/`, in the same way as the magnet would be changed by the user

do you still think it should be inlined?

Reply:
They can still change it, but for the sake of the tutorial, where the variable is only used once in a single place, it should be inlined.

Reply:
> They can still change it, but for the sake of the tutorial, where the variable is only used once in a single place, it should be inlined.

its done the same for the magnet tho, which is also used once, imho its better to keep it not inline

Reply:
magnet is long tho and in a real app it will come from a variable